test/fixtures/math_spec.tsx

```diff
@@ -0,0 +1,9 @@
+import math from './math'
```

Comment:
This isn't actually a spec. Maybe the contents should be `math.tsx` and then `math_spec.tsx` could actually test it.

Reply:
I included enzyme test.